angular/src/directives/proxies.ts

```diff
@@ -31,7 +31,10 @@ export class IonAccordion {
 import type { AccordionGroupChangeEventDetail as IAccordionGroupAccordionGroupChangeEventDetail } from '@ionic/core';
 export declare interface IonAccordionGroup extends Components.IonAccordionGroup {
   /**
-   * Emitted when the value property has changed. 
+   * Emitted when the value property has changed
```

Comment:
This was auto-generated. The formatting looks weird, but should be fine?

Reply:
Yeah this should be fine. Intellisense still formats the text correctly. Just a quirk of the output target generator (we would have to split on line breaks and generate per line - which is possible, but the raw output works as is for now).